README.md

```diff
@@ -7,32 +7,43 @@
+ElectrsCash is an efficient implementation of Electrum Server and can be used
+as a drop-in replacement for ElectrumX. In addition to the TCP RPC interface,
+it also provides WebSocket support.
 
-ElectrsCash extends the original Electrum protocol, supporting additional
-technology well established in the ecosystem such as CashAccounts.
+ElectrsCash fully implements the
+[v1.4.3 Electrum Cash protocol](https://bitcoincash.network/electrum/)
+and in addition to [useful extensions](doc/rpc.md), including CashAccounts.
 
-The server indexes the entire Bitcoin Cash blockchain, and the resulting index enables fast queries for any given user wallet,
-allowing the user to keep real-time track of his balances and his transaction history using the [Electron Cash wallet](https://electroncash.org/).
-Since it runs on the user's own machine, there is no need for the wallet to communicate with external Electrum servers,
+The server indexes the entire Bitcoin Cash blockchain, and the resulting index
+enables fast queries for blockchain applications and any given user wallet,
+allowing the user to keep real-time track of his balances and his transaction
+history.
+
+When run on the user's own machine, there is no need for the wallet to
+communicate with external Electrum servers,
 thus preserving the privacy of the user's addresses and balances.
 
 ## Features
 
- * Supports Electrum protocol [v1.4.2](https://bitcoincash.network/electrum/)
- * Maintains an index over transaction inputs and outputs, allowing fast balance queries
- * Fast synchronization of the Bitcoin Cash blockchain on modest hardware
- * Low index storage overhead (~20%), relying on a local full node for transaction retrieval
- * `txindex` is not required for the Bitcoin node, however it does improve performance
- * Uses a single [RocksDB](https://github.com/spacejam/rust-rocksdb) database, for better consistency and crash recovery
-
-## Notable features unique to ElectrsCash
-
- * [CashAccount support](https://honest.cash/v2/dagur/fast-cashaccount-lookups-using-bitbox-and-electrum-4781)
- * Supports major Bitcoin Cash full nodes, in addition to [full integration with Bitcoin Unlimited](https://github.com/BitcoinUnlimited/BitcoinUnlimited/blob/release/doc/bu-electrum-integration.md)
- * Deterministic builds
- * We're the only electrum server with good integration tests coverage (see Tests below)
+- Supports Electrum protocol [v1.4.3](https://bitcoincash.network/electrum/)
+- Maintains an index over transaction inputs and outputs, allowing fast balance
+  queries.
+- Fast synchronization of the Bitcoin Cash blockchain on modest hardware
+- Low index storage overhead (~20%), relying on a local full node for
+  transaction retrieval.
+- `txindex` is not required for the Bitcoin node, however it does improve
+  performance.
+- Uses a single [RocksDB](https://github.com/spacejam/rust-rocksdb) database
+  for better consistency and crash recovery.
+
+- Notable features unique to ElectrsCash
+
```

Comment:
add one-level indentation for the following list? (since they are sub items of item above)